Update on COVID-19 (#1190)
writethedocs/www: docs/conf/portland/2020/news/covid-19-update-01.rst

:template: {{year}}/generic.html

.. post:: March 3, 2020
   :tags: portland-2020, covid-19

COVID-19 Update
===============

Hey Folks,

> As you've probably heard by now, the coronavirus (also known as COVID-19) is a new virus that causes respiratory illness in people and can easily spread from person-to-person. Since Write the Docs Portland 2020 is only a few months away, we wanted to give you an update on our status regarding COVID-19.

Reviewer: ```suggestion
As you've probably heard by now, the coronavirus (also known as COVID-19) is a new virus that causes respiratory illness in people and can easily spread from person-to-person. Since Write the Docs Portland 2020 is only 2 months away, we wanted to give you an update.
```

**As of today, March 3, Write the Docs Portland 2020 is scheduled to go ahead**.

> We are watching the situation closely, and taking advice from the Oregon health authorities and the United States CDC (Center for Disease Control). We plan to reassess the situation weekly, you can find updates tagged with `COVID-19 </blog/archive/tag/covid-19/>`_.

Reviewer 2: ```suggestion
We are watching the situation closely, and taking advice from the Oregon health authorities and the United States CDC (Centers for Disease Control and Prevention). We plan to reassess the situation weekly, you can find updates tagged with `COVID-19 </blog/archive/tag/covid-19/>`_.
```

Author: CDCP did you say? @ddbeck :japanese_ogre:

Reviewer 2: Hey, of all the weird things about US government, this initialism mismatch is the least weird

> If at the time of Write the Docs Portland 2020 you feel sick, or are worried that you might have been in contact with people who have been diagnosed with COVID-19, we encourage you to stay home. Your registration will be 100% refunded. If you have any questions about this, please reach out to `{{email}} <mailto:{{email}}>`_.

Reviewer: ```suggestion
If at the time of Write the Docs Portland 2020 you feel sick, or are worried that you might have been in contact with people who have been diagnosed with COVID-19, we encourage you to stay home. Your registration will be 100% refunded. If you have any questions about this, please reach out to `{{email}} <mailto:{{email}}>`_. We will continue to provide updates as the conference gets closer.
```

If at the time of Write the Docs Portland 2020 you feel sick, or are worried that you might have been in contact with people who have been diagnosed with COVID-19, we encourage you to stay home. Your registration will be 100% refunded. If you have any questions about this, please reach out to `{{email}} <mailto:{{email}}>`_.

> We will continue to provide updates as the conference gets closer. In the meantime, we recommend watching these resources that will effect our decisions:

Author: ```suggestion
We will continue to provide updates as the conference gets closer. In the meantime, we recommend keeping an eye on the official resources at:
```

Author: @ericholscher  WDYT?
